index.html

```diff
@@ -132,5 +132,29 @@ <h3>LETS CONNECT</h3>
         </ul>
     </div>
 </section>
+<section class="contact-form">
+    <h1>Contact me</h1>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form method="" action="">
```

Comment:
kindly fill the method and action attributes as requested in this project requirements.